app/models/user.rb

```diff
@@ -10,9 +11,33 @@ class User < ApplicationRecord
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
 
-  # 渡された文字列のハッシュ値を返す
-  def User.digest(string)
-    cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST : BCrypt::Engine.cost
-    BCrypt::Password.create(string, cost: cost)
+  class << self # この定義方法よくわからない
```

Comment:
http://qiita.com/Linda_pp/items/b7135ae1f0def6058c6c#%E8%A4%87%E6%95%B0%E5%AE%9A%E7%BE%A9%E3%81%99%E3%82%8B%E6%99%82